EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -9978,7 +9978,7 @@ QUEST_LV_0200_20151224_009981	Not to mention that cannon balls cannot be compare
 QUEST_LV_0200_20151224_009982	I'm not just trying to tell you how hard it is for me, though.
 QUEST_LV_0200_20151224_009983	Isn't the markmanship of Musketeers just fascinating? With skills like that....{nl}You can fire fast and clear a whole group of enemies or step up and deliver that fatal blow.
 QUEST_LV_0200_20151224_009984	After Medzio Diena, Plague Doctors became more important than ever.{nl}That's why I want to mentor future Plague Doctors and help the world.
-QUEST_LV_0200_20151224_009985	I know many people think numbers are difficult.{nl}I really don't know how something so simple and clear can be difficult, though.
+QUEST_LV_0200_20151224_009985	I know many people think numbers are difficult.{nl}I really don't know how something so simple and clear can be difficult.
```

Comment:
`I know many people think numbers are difficult,{nl}but I really don't know how something so simple and clear can be difficult.` You can probably leave it as is, but I feel like these sentences flow better when connected like this